spec/requests/api/v1/moves_controller_filter_spec.rb

```diff
@@ -29,27 +29,46 @@
       end
 
       describe 'by created_at' do
-        let(:first_date) { Date.new(2019, 12, 25) }
-        let(:last_date) { Date.new(2019, 12, 27) }
-        let(:middle_date) { Date.new(2019, 12, 26) }
+        let(:first_date) { DateTime.new(2019, 12, 25, 12) }
+        let(:last_date) { DateTime.new(2019, 12, 27, 12) }
+        let(:middle_date) { DateTime.new(2019, 12, 26, 12) }
 
         let(:target_move) { Move.find_by(created_at: first_date) }
         let(:target_move2) { Move.find_by(created_at:  middle_date) }
         let(:target_move3) { Move.find_by(created_at:  last_date) }
-        let(:filter_params) { { filter: { created_at_from: first_date.to_s, created_at_to: last_date.to_s } } }
         let(:all_dates) { [first_date, middle_date, last_date] }
 
         before do
           create(:move, created_at: first_date)
           create(:move, created_at: middle_date)
           create(:move, created_at: last_date)
-        end
 
-        it 'returns the right amount of moves' do
           get '/api/v1/moves', headers: headers, params: filter_params
+        end
+
+        context 'when happy case' do
```

Comment:
would be nice to add a test that checks the created_at_from/to filter actually works

Reply:
there is one - this is a very small big fix to that PR

Reply:
I'm sorry - I don't see the filter test you are referring too? can you point me in the right direction